integration/eth2network/eth2_network.go

```diff
@@ -321,22 +322,36 @@ func (n *Impl) Start() error {
 // Stop stops the network
 func (n *Impl) Stop() error {
 	for i := 0; i < len(n.dataDirs); i++ {
-		err := n.gethProcesses[i].Process.Kill()
+		err := kill(n.gethProcesses[i].Process, 0)
 		if err != nil {
 			fmt.Printf("unable to kill geth node - %s\n", err.Error())
-			return err
 		}
-		err = n.prysmBeaconProcesses[i].Process.Kill()
+		err = kill(n.prysmBeaconProcesses[i].Process, 0)
 		if err != nil {
 			fmt.Printf("unable to kill prysm beacon node - %s\n", err.Error())
-			return err
 		}
-		err = n.prysmValidatorProcesses[i].Process.Kill()
+		err = kill(n.prysmValidatorProcesses[i].Process, 0)
 		if err != nil {
 			fmt.Printf("unable to kill prysm validator node - %s\n", err.Error())
-			return err
 		}
 	}
+	// wait a second for the kill signal
+	time.Sleep(time.Second)
+	return nil
+}
+
+const maxTryKill = 5
+
+func kill(p *os.Process, cnt int) error {
```

Comment:
I tried to make the kill a bit more robust.
No longer stopping at the first error